From ccd9bb7936f4e9bcb41179c8ff4ba96d1a204591 Mon Sep 17 00:00:00 2001
From: "Shawn W. Dunn" <sfalken@cloverleaf-linux.org>
Date: Thu, 16 Jan 2025 10:39:28 -0800
Subject: [PATCH 3/3] add option for miriway

Miriway is a new generic compositor built on the Mir compositor library that
is designed to be used as part of composable desktop environments.

See: https://github.com/lxqt/lxqt-session/pull/571
---
 config/waylandwindowmanagers.conf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/config/waylandwindowmanagers.conf b/config/waylandwindowmanagers.conf
index e727f43..e0ddbb2 100644
--- a/config/waylandwindowmanagers.conf
+++ b/config/waylandwindowmanagers.conf
@@ -9,6 +9,9 @@ kwin_wayland/Comment=Window manager of the KDE Software Compilation
 labwc/Name=Labwc
 labwc/Comment=wlroots-based window-stacking compositor, inspired by openbox
 
+miriway/Name=Miriway
+miriway/Comment=Simple Wayland compositor built on Mir for building desktop environments
+
 niri/Name=niri
 niri/Comment=A scrollable-tiling Wayland compositor
 
-- 
2.48.0

